frontend/src/views/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,9 @@
+import { TopBar } from 'components/TopBar/TopBar';
+
+export const TodoList = ({ pageTitle }: { pageTitle: string }) => {
+  return (
+    <>
+      <TopBar pageTitle={pageTitle} />
+    </>
+  );
+};
```

Comment:
A może damy radę skrócić?

```suggestion
import { TopBar } from 'components/TopBar/TopBar';

export const TodoList = ({ pageTitle }: { pageTitle: string }) => {
  return (<TopBar pageTitle={pageTitle} />);
};
```

Albo jeszcze bardziej?
```suggestion
import { TopBar } from 'components/TopBar/TopBar';

export const TodoList = ({ pageTitle }: { pageTitle: string }) => <TopBar pageTitle={pageTitle} />
```